rector.php

```diff
@@ -24,6 +26,10 @@
 
         // buggy when remove private property, but filled by construct in the middle of parameter
         RemoveUnusedPrivatePropertyRector::class,
+
+        RemoveUnusedPrivateMethodRector::class => [
+            ConfigurationBootloader::class,
+        ],
```

Comment:
keep private method `ConfigurationBootloader::configManager()` on purpose as it used in singleton:

https://github.com/spiral/framework/blob/cc6cb37254e8185561f12c9e6d1c0ade824ff9a3/src/Boot/src/Bootloader/ConfigurationBootloader.php#L33